paws_collector.js

```diff
@@ -153,55 +154,25 @@ class PawsCollector extends AlAwsCollector {
             function(asyncCallback) {
                 return collector.pawsGetLogs(pawsState.priv_collector_state, asyncCallback);
             },
-            function(logs, privCollectorState, nextInvocationTimeout, asyncCallback) {
+            function(logs, newExtState, nextInvocationTimeout, asyncCallback) {
                 console.info('PAWS000200 Log events received ', logs.length);
                 return collector.processLog(logs, collector.pawsFormatLog, null, function(err) {
-                    return asyncCallback(err, privCollectorState, nextInvocationTimeout);
+                    return asyncCallback(err, newExtState, nextInvocationTimeout);
                 });
             },
-            function(privCollectorState, nextInvocationTimeout, asyncCallback) {
-                return collector._storeCollectionState(pawsState, privCollectorState, nextInvocationTimeout, asyncCallback);
+            function(newExtState, nextInvocationTimeout, asyncCallback) {
+                return collector._storeCollectionState(pawsState, newExtState, nextInvocationTimeout, asyncCallback);
             }
         ], function(error) {
             collector.done(error);
         });
-    };
-    
-    _storeCollectionState(pawsState, privCollectorState, invocationTimeout, callback) {
-        if (Array.isArray(privCollectorState)) {
-            return this._storeCollectionStateArray(pawsState, privCollectorState, invocationTimeout, callback);
-        } else {
-            return this._storeCollectionStateSingle(pawsState, privCollectorState, invocationTimeout, callback);
-        }
     }
     
-    _storeCollectionStateArray(pawsState, privCollectorStates, invocationTimeout, callback) {
```

Comment:
@OHaganA looks like your changes are not rebase on top of the latest ones.

Reply:
Ah, whoops, the problems of a quick 5 minute PR, fixing